docs: add advanced theme configuration option to setup guide (#284)
VerifiedInc/VerifiedInc.github.io: versioned_docs/version-2.1/1-click-signup/guides/setup.mdx

        <td>4</td>
        <td>Border radius used for all buttons</td>
    </tr>
    <tr>
        <td>Advanced Theme</td>
        <td>Custom JSON</td>
        <td>-</td>

>         <td>Custom JSON configuration that allows fine-grained control over any SDK component styling. It laverages MUI's theme system. [Learn more](https://mui.com/material-ui/customization/default-theme)</td>

Reviewer: ```suggestion
        <td>Custom JSON configuration that allows fine-grained control over any SDK component styling, using the [MUI theme system](https://mui.com/material-ui/customization/default-theme)</td>
```

Reviewer: Typo fix and rewording for clarity
